lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -301,17 +301,11 @@ html.video-fullscreen {
           &.fa-arrow-circle-right {
             top: 30%;
           }
-        }
-        &:last-child {
-          > .fa {
+          &.exam-without-proctoring {
```

Comment:
Thanks.  I forgot about the other PR. :)  👍  with this change.  It's too bad this can't live with its reference.